RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp

```diff
@@ -120,6 +120,11 @@ namespace Ogre {
                                 "EGLWindow::create");
                 }
 
+                if((opt = miscParams->find("currentEGLSurface")) != end)
```

Comment:
also add the parseBool part, as currently `currentEGLSurface=false` would be accepted

Reply:
@paroj 
Thanks,
I've added